### PR TITLE
fix #13727 by removing <small> elements which contains the number of ordered by columns"

### LINKED ISSUE #13727: Copied table data add "1" after column name which had order by parameter
### Steps to reproduce
1. Select any table 
2. order by any column
3. from menu/option in bottom on table select "copy to clipboard"

### Expected behaviour
"1" do not depict the why it is used for
Maybe we can add [Asc] [Desc] or anything else or  nothing

### Actual behaviour
"1" is added after column name which had order by parameter
copy

### Server configuration
Online demo

**phpMyAdmin version:** 4.7.4

### Client configuration
**Browser:** 61.0.3163.100 (Official Build) (64-bit)

**Operating system:** Ubuntu 16.0.4 LTS

### PR DESCRIPTION
**Pull request by xypXiong**
Before submitting pull request, please check that every commit:

- [ ] Has proper Signed-Off-By
- [ ] Has commit message which describes it
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

File: js/sql.js [hunk 1]
```diff
@@ -286,7 +286,8 @@ AJAX.registerOnload('sql.js', function () {
         });
 
         $('.table_results .column_heading a').each(function () {
-            textArea.value += $(this).text() + '\t';
+        	//remove <small> elements which contains the number of ordered by columns        
+        	textArea.value +=($(this).remove("small")).text() + '\t';
```
Comment (nijel) [inline js/sql.js]: Doesn't this remove the elements from the page? So that they are no longer visible after copying the text?

**Comment by xypXiong** [comment]
no, it does not.  only make the copy  function not showing the ordering number. please give a try.
My comment is not that clear.
Thanks,Yanping 

    Michal Čihař <notifications@github.com> 于 2017年11月16日, 星期四, 上午 5:24 写道：
 

 @nijel commented on this pull request.In js/sql.js:> @@ -286,7 +286,8 @@ AJAX.registerOnload('sql.js', function () {
         });
 
         $('.table_results .column_heading a').each(function () {
-            textArea.value += $(this).text() + '\t';
+        	//remove <small> elements which contains the number of ordered by columns        
+        	textArea.value +=($(this).remove("small")).text() + '\t';        	
Doesn't this remove the elements from the page? So that they are no longer visible after copying the text?—

**Comment by xypXiong** [comment]
Sorry, just found the line has a problem. Will check in a fix soon.
Thanks,Yanping 

    Michal Čihař <notifications@github.com> 于 2017年11月16日, 星期四, 上午 5:24 写道：
 

 @nijel commented on this pull request.In js/sql.js:> @@ -286,7 +286,8 @@ AJAX.registerOnload('sql.js', function () {
         });
 
         $('.table_results .column_heading a').each(function () {
-            textArea.value += $(this).text() + '\t';
+        	//remove <small> elements which contains the number of ordered by columns        
+        	textArea.value +=($(this).remove("small")).text() + '\t';        	
Doesn't this remove the elements from the page? So that they are no longer visible after copying the text?—

**Comment by nijel** [comment]
I guess this is superseded by #13823.

**Pull request closed without merging**